### ISSUE TITLE
"View All" link for Places is missing on web /explore page

### ISSUE THREAD

**Issue by bhugh** [immich-app/immich#6699]
### The bug

In the "Explore" tab of the web version of Immich, the "Places" list is missing a "View All" link.  So you can see the first few places alphabetically but there is no way to view all the rest of the places.

The "View All" button is available in the Android app and works just fine.  It looks like the "View All" link was just inadvertently omitted in the web version.

I see this in both my own immich server running from home and also the demo server running on demo.immich.app.

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

v1.93.3

### Version of Immich Mobile App

(not relevant"

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
(Not relevant to this issue)
```

### Your .env content

```Shell
(Not relevant to this issue)
```

### Reproduction steps

```bash
1.Start the server
2.Open the web app and look at the /explore page - ie https://demo.immich.app/explore
3.Note that there is a "View All" link for "People" but no "View All" link for "Places".
```

### Additional information

I tried this is both Edge and Chrome just now - both are the same.  I have noticed this link missing for a while in different environments (different computers, web browser on my phone, etc), so I believe it is actually missing, not just some specific bug related to a particular browser or setup.

**Comment by bo0tzz** [comment]
Recently fixed in #6669

**Comment by aisbergde** [comment]
@bo0tzz please reopen the issue. The "view all" button is there, but there are still only the first 12 places shown (v1.94.1):
![image](https://github.com/immich-app/immich/assets/9020187/5149ffa4-4c0b-4181-8906-11b2c1876c1c)
